patches/nodejs-mobile-react-native+18.17.7+001+jcenter-to-maven.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/nodejs-mobile-react-native/android/build.gradle b/node_modules/nodejs-mobile-react-native/android/build.gradle
+index 705e4c8..cc1034f 100644
+--- a/node_modules/nodejs-mobile-react-native/android/build.gradle
++++ b/node_modules/nodejs-mobile-react-native/android/build.gradle
+@@ -1,11 +1,11 @@
+ 
+ buildscript {
+     repositories {
+-        jcenter()
++        mavenCentral()
+     }
+ 
+     dependencies {
+-        classpath 'com.android.tools.build:gradle:2.2.3'
++        classpath 'com.android.tools.build:gradle:2.3.0'
+     }
+ }
```

Comment:
nit: add an empty line at the end (consistent with how the other patch files are formatted)